.github/workflows/update-docker-requirements.yml

```
@@ -42,10 +43,22 @@ jobs:
           NEEDS_UPDATE=$(git diff --quiet && echo 'false' || echo 'true')
           echo "needs-update=${NEEDS_UPDATE}" | tee --append $GITHUB_OUTPUT
 
+      - name: Get last merged PR info
+        id: pr_info
```

Let's name this
 
```suggestion
        id: pr-info
```

as we do everywhere else in this repo as well.